infra/aws/terraform/prow-build-cluster/resources/flux-system/slack-alerting-external-secret.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: slack-url
+  namespace: flux-system
+spec:
+  refreshInterval: 1h
+  secretStoreRef:
+    name: aws-secerts-manager
+    kind: ClusterSecretStore
+  target:
+    name: slack-url
+    creationPolicy: Owner
+  data:
+  - secretKey: address
```

Comment:
Can we call this secretKey `url` or is it required by Flux to be `address`?

Reply:
Oh, I just realised that this is just a regular k8s secret ref, so I can actually change the key.